GitFyle.Core.Api/Models/Foundations/Repositories/Repository.cs

```diff
@@ -0,0 +1,31 @@
+﻿using GitFyle.Core.Api.Models.Foundations.Sources;
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace GitFyle.Core.Api.Models.Foundations.Repositories;
+
+public class Repository
+{
+    public Guid Id { get; set; }
+
+    [Required]
+    [MaxLength(255)]
+    public string Name { get; set; }
```

Comment:
Let's remove validations at the model level they should be here. As annotations. Let's handle this in Service/Foundation layer and the fluent rule enforcement at the configuration level.